### PR TITLE
[CI] Restore release assets

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,7 +1,5 @@
 name: OctoBot-Binary-CI
 on:
-  schedule:
-    - cron: '0 5 * * 1' # every monday at 5am
```
Comment (GuillaumeDSM) [inline .github/workflows/main.yml:4]: why removing the cron job ?

**Review by GuillaumeDSM — COMMENTED** [review]
looks great !
did you manage it test it ? ^^

**Comment by Herklos** [comment]
> looks great ! did you manage it test it ? ^^

How would you like me to have tested it? It's not possible without releasing :/

File: .github/workflows/main.yml (hunk 1 above)
Comment (Herklos) [reply .github/workflows/main.yml:4]: Because it cannot be used anymore if we don't push to an external storage.
Otherwise, we can build it for fun.

Comment (GuillaumeDSM) [reply .github/workflows/main.yml:4]: the idea is also to check that builds keep working, I think we should keep it and not upload binaries

**Comment by GuillaumeDSM** [comment]
> > looks great ! did you manage it test it ? ^^
> 
> How would you like me to have tested it? It's not possible without releasing :/

right :/

File: .github/workflows/main.yml (hunk 1 above)
Comment (Herklos) [reply .github/workflows/main.yml:4]: ok

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**